yeast-core/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "yeast-core",
-	"version": "1.1.1",
+	"version": "1.1.2",
```

Comment:
bump the minor version instead (i.e. 1.2.0) since this contains a significant change. It's technically a breaking change that should have a major version bump, but I don't want to raise the main version. (containsTextModification renamed to containsDiff)